src/components/Spinner/readme.md

```diff
@@ -115,3 +115,37 @@ const Loading = styled.h1.attrs(props => {
     </Loading>
 </InverseContainer>
 ```
+
+##### list loading using spinner - brand - large - arc
```

Comment:
I think this example names are missleading, there is nothing related to lists here. I am thinking about using something like `Arc Spinner large brand`; same for the rest of the exampe headers